templates/details_base.html

```diff
@@ -75,6 +75,30 @@ <h2 class="govuk-heading-s govuk-!-margin-top-3">
                 {{entity.custom_properties.data_summary.refresh_period}}
               </li>
             {% endif %}
+            {% if entity.custom_properties.audience %}
+              <li>
+                <span class="govuk-!-font-weight-bold">Audience:</span>
+                {{entity.custom_properties.audience}}
+              </li>
+            {% endif %}
+            {% if entity.custom_properties.provider %}
+              <li>
+                <span class="govuk-!-font-weight-bold">Metadata provider:</span>
+                {{entity.custom_properties.provider}}
+              </li>
+            {% endif %}
+            {% if entity.last_updated %}
+              <li>
+                <span class="govuk-!-font-weight-bold">Data last updated:</span>
+                {{entity.last_updated|date:"d M Y"}}
+              </li>
+            {% endif %}
+            {% if entity.last_modified %}
```

Comment:
#### UX suggestion

"Metadata last updated" feels really out of place here to me - in my mind it's metadata *about metadata* (or data about data about data) whereas everything else in this list is about *the data itself*.

I'm wondering if we could show this in a different way that visually suggests what it applies to, e.g.

![mockup with metadata last updated in the bottom right of the panel](https://github.com/user-attachments/assets/f290df50-702b-407d-8a11-6c3b3b33791c)

We know the concept of metadata is going to confuse a lot of users, so I don't think we should rely on users being able to tell the difference based on labels alone.

Reply:
I like this idea, I'll implement it 🚀